lib/matplotlib/ticker.py

```diff
@@ -623,7 +623,7 @@ def set_powerlimits(self, lims):
 
     def format_data_short(self, value):
         # docstring inherited
-        if isinstance(value, np.ma.MaskedArray) and value.mask:
+        if isinstance(value, np.ma.core.MaskedConstant):
```

Comment:
I haven’t completely followed where `value` comes from, but would it be enough to check
```suggestion
        if value is np.ma.masked:
```
?
https://numpy.org/doc/stable/reference/maskedarray.baseclass.html#numpy.ma.masked

Reply:
`value` is just a single value, although it may come from a masked array.

Yes, that would be the same thing.